public/js/utils/pretty-print.js

```diff
@@ -0,0 +1,37 @@
+const { getValue } = require("../feature");
+const { workerTask } = require("./utils");
+const { isJavaScript } = require("./source");
+const assert = require("./assert");
+
+let prettyPrintWorker;
+prettyPrintWorker = new Worker(
```

Comment:
Yep, that was leftover from the ability to restart it (which we don't need here, it doesn't keep any internal state)